src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutesSpec.scala

```diff
@@ -11,18 +11,24 @@ import spray.json._
 
 class LeoRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest with TestLeoRoutes with TestComponent {
 
+  private val googleProject = "test-project"
```

Comment:
-1 on
```
private val foo = "string"
private val fooObject = Something(foo)
```
Just create the objects and when strings are needed unwrap with `.string`